recipe/recipe.yaml

```diff
@@ -43,8 +56,13 @@ tests:
         - ${{ "Library/" if win }}share/renode-cli/platforms/cpus/cortex-a9.repl
         - ${{ "Library/" if win }}share/renode-cli/tools/sel4_extensions/seL4Extensions.cs
   - script:
-      - renode --help
-      - LOCAL_TEST_PREFIX=./test-bundle test-bundle/${{ "Library/" if win }}bin/renode-test test-bundle/${{ "Library/" if win }}share/renode-cli/tests/unit-tests
+      - if: target_platform == "linux-aarch64"
+        then:
+          - DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1 renode --help
+          - DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1 LOCAL_TEST_PREFIX=./test-bundle test-bundle/${{ "Library/" if win }}bin/renode-test test-bundle/${{ "Library/" if win }}share/renode-cli/tests/unit-tests
```

Comment:
We probably need `icu` as a run-time dependency instead of setting `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT`.

Reply:
I think that's what I had before. I tried to use CLR_xxx to set the version as well, but there seems to be something that prevents to load it

Reply:
dotnet-runtime has it icu as runtime dependency and it is in the test environment:
```
 │ │ dotnet-runtime        ┆ 8.0.14       ┆ h3250cb0_1        ┆ conda-forge ┆  24.76 MiB │
 │ │ icu                   ┆ 75.1         ┆ hf9b3779_0        ┆ conda-forge ┆  11.71 MiB │
 │ │ keyutils              ┆ 1.6.1        ┆ h4e544f5_0        ┆ conda-forge ┆ 109.69 KiB │
```

Reply:
In fact I proposed a PR to update the activation on aarch64